src/main/webapp/queue.js

```diff
@@ -63,7 +63,7 @@ function updateShareTab() {
       '<span id="shareImg" class="material-icons">content_copy</span>' +
       '</button>' +
       '</div>' +
-      '<input id="loungeLink" value="https://www.youtube-lounge.appspot.com/' +
+      '<input id="loungeLink" value="http://www.youtube-lounge.appspot.com/' +
```

Comment:
why are we moving to http?

Reply:
Also I think you could just replace it to relative path /lounge.html and it should work without the server hostname. that way you don't need to edit the Html when the server hostname changes.

Reply:
Moving to http to avoid issues with unnecessary protections on API key that prevent access from multiple clients. I wrote the entire link so that the user can send the entire link to their friends instead of having to append the host name themselves. I don't think changing it reveals any benefit. Let me know what you think.

Reply:
Also, the server hostname shouldn't ever change since the link we're provided is stagnant and the room parameters are always passed in by way of the url, so it's not possible to incorrectly load the url into the input for copying

Reply:
Is the link generated by the javascript?  can't you do 

window.location.hostname + (existing logic to get the loungelink value)?

I mean the hostname can change if you plan to create a new GCP project right? then you'd have to manually edit the HTML files.